app/views/two_factor_authentication/webauthn_verification/show.html.erb

```diff
@@ -28,7 +28,7 @@
       </div>
     </div>
     <p class="half-center">
-      <%= t('forms.webauthn_setup.login_text') %>
+      <input type="button" id="webauthn-button" class="usa-button usa-button--big usa-button--wide" value="<%= t('links.continue_sign_in') %>">
```

Comment:
I think the page is slightly confusing now. The animation makes it feel like things are going on, and "Continue sign in" isn't super clear to me. Is there other language we can use? "Press to authenticate with your security key" or something?

![image](https://user-images.githubusercontent.com/1430443/122795786-47146600-d283-11eb-989b-8e6054188ba7.png)

Reply:
How about removing the animation?

Reply:
Scratch my last comment.  When the user clicks the button the browser automatically creates a browser specific popup which tells them to insert their key and press the button.

Reply:
The browser specific popup tells them to insert the key and press the button to continue
![Screen Shot 2021-06-21 at 11 57 28 PM](https://user-images.githubusercontent.com/34244063/122865755-7f9c5a00-d2ec-11eb-97c4-0a1e98850ddc.png)
![Screen Shot 2021-06-21 at 11 54 33 PM](https://user-images.githubusercontent.com/34244063/122865762-82974a80-d2ec-11eb-800e-4941412af286.png)

Reply:
Regarding wording, “button” is probably an important term to avoid since it could be read as referring to the button on a hardware token as well as a button on the web page. 

GitHub uses a more specific “Use security key" message:

![Screenshot 2021-06-22 at 15-26-54 Build software better, together](https://user-images.githubusercontent.com/46565/122987355-823f9380-d36e-11eb-884f-c0349856bfa6.png)

Google appears to avoid this entirely by triggering the login flow in the event handler for the password submit button so it already has the user-initiated trigger context and can open the browser's native dialog without an intermediate step.